spec/spec.md

```diff
@@ -457,11 +457,57 @@ The following properties are for use at the top-level of the resource — all
 other properties that are not defined below MUST be ignored:
 
 - `name` - The resource ****MAY**** contain this property, and if present its
-value ****SHOULD**** be a human-friendly name that describes what the
-Presentation Definition pertains to.
+  value ****SHOULD**** be a human-friendly name that describes what the
+  Presentation Definition pertains to.
 - `purpose` - The resource ****MAY**** contain this property, and if present its
-value ****MUST**** be a string that describes the purpose for which the
-Presentation Definition's inputs are being requested.
+  value ****MUST**** be a string that describes the purpose for which the
+  Presentation Definition's inputs are being requested.
+- The resource ****MAY**** include a `format` property, and its value 
+  ****MUST**** be an object with one or more properties matching the registered 
+  [Credential Format Designations](#credential-format-designations) to inform the 
+  Holder of the credential format configurations the Verifier can process. The 
+  value for each property included ****MUST**** be an object composed as follows:
+    - The object ****MAY**** include a format-specific property (i.e. `alg`, `type`) 
+      that expresses which algorithms the Verifier supports for the format, and if 
+      present, its value ****MUST**** be an array of one or more of the 
+      format-specific algorithmic identifier references, as noted in the 
+      [Credential Format Designations](#credential-format-designations) section.
+
+      ```json
+      {
+        "presentation_definition": {
+          "format": {
+            "jwt": {
+              "alg": ["EdDSA", "ES256K", "ES384", "PS256"]
+            },
+            "jwt-vc": {
+              "alg": ["EdDSA", "ES256K", "ES384", "PS256"]
+            },
+            "jwt-vp": {
+              "alg": ["EdDSA", "ES256K", "ES384", "PS256"]
+            },
+            "vc": {
+              "type": [
```

Comment:
suggest we rename this to `proofType` to avoid confusion

Reply:
@OR13 noted that the types aren't exclusively used for proofs, type is more inclusive and aligns with ld sigs

Reply:
Yes, it aligns with LD proof objects, but its proximity to "vc" and "vp" *as a child element* strongly misdirect the casual reader

Reply:
yep, i can see that... vp and vc have their own `type` member... better to be more explicit.